Source/FakeItEasy.Specs/ConstructorWithVirtualCallsSpecs.cs

```diff
@@ -106,16 +260,9 @@ public FakedClass()
             {
                 this.StringPropertyValueDuringConstructorCall = this.StringProperty;
                 this.ValueTypePropertyValueDuringConstructorCall = this.ValueTypeProperty;
-                this.InterfacePropertyMethodResultDuringConstructorCall = this.InterfaceProperty.InnerProperty;
 
                 this.StringProperty = "value set in constructor";
                 this.ValueTypeProperty = 123456;
-                this.InterfaceProperty.InnerProperty = "value set in constructor";

```

Comment:
The idea of the `InterfaceProperty`-tests was to prove even the fake *creation*+configuration works within virtual c'tor calls (using an virtual interface property as an example). Is this not a valid corner case?